main.py

```diff
@@ -89,6 +89,7 @@ def main(start_date: str, open_in_browser: bool):
     )
     parser.add_argument(
         "--open_in_browser",
+        action="store_true",
```

Comment:
Is it normal to specify both `store_true` and a default of False? Aren't they functionally equivalent in this case?

If they are, I prefer `default` - I find the `store_true` and `store_false` functionality are less straight forward to understand.

I don't write many CLIs - please let me know if I'm missing some nuance with this.

Reply:
Sorry, I have been away from my computer for the past couple of days. I went with ```store_true``` because that was the first one that solved the issue, but ```default``` works as well.

Reply:
Sorry for the late reply, I have been away from my computer for the past couple of days. I checked and ```default``` works. I went with ```store_true``` because it was the first thing that I saw in the documentation that fixed the problem.